data/brands/shop/sports.json

```
@@ -93,6 +93,17 @@
         "sport": "golf"
       }
     },
+    {
+      "displayName": "Arei Outdoor Gear",
+      "id": "areioutdoorgear-f9c896",
+      "locationSet": {"include": ["id"]},
+      "tags": {
+        "brand": "Arei Outdoor Gear",
+        "brand:wikidata": "Q54974273",
```

```suggestion
        "brand:wikidata": "Q114343299"
```
Most likely a copy-paste error